### PR TITLE
Update/circular progress bar props

### PR DESCRIPTION
**Pull request by agrullon95**
#### Testing/Review estimation

Testing: short
Review: short

#### Proposed Changes

Update the Circular progress bar component to use a `size` prop (changing the size of the component) and `enableMobileScaling` prop to apply mobile view port styling.

#### Testing Instructions

* Checkout this branch
* Create a new launchpad site or use an existing site.
* Navigate to the launchpad screen and verify the Circular Progressbar matches production.

image

#### Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

Related to #

### CODE REVIEW

**Comment by sharpirate** [comment]
Changes look good and it tests well:

✔️ Style looks good on desktop
✔️ Style looks good on mobile

**Review by sharpirate — APPROVED** [review]

File: packages/components/src/circular-progress-bar/index.tsx [hunk 1]
```diff
@@ -1,21 +1,28 @@
+import classnames from 'classnames';
 import './style.scss';
 
 const CircularProgressBar = ( {
 	currentStep,
 	numberOfSteps,
+	size,
```
Comment (edanzer) [inline packages/components/src/circular-progress-bar/index.tsx:7]: Is there any value in making `size` optional with a default value?

**Comment by edanzer** [comment]
This is testing well for me, and changes look straightforward.

**Review by edanzer — APPROVED** [review]

File: packages/components/src/circular-progress-bar/index.tsx (hunk 1 above)
Comment (agrullon95) [reply packages/components/src/circular-progress-bar/index.tsx:7]: We could make `size` optional, but I also wanted to ensure that developers always set a size for this component.

**Pull request merged**